eia_app/forms.py

```diff
@@ -0,0 +1,39 @@
+'''Forms para el uso del crud del consultor'''
+
+from django import forms
+from .models import * #pylint: disable=unused-wildcard-import, wildcard-import
```

Comment:
Es mejor que importes lo que necesites en vez de importar todo
```suggestion
from .models import (
    Organizacion, Responsable, Solicitante,
    DatosProyecto, DatosDocumento
)
```